### PR TITLE
KAA-1309 Minor inaccuracy on C++/SDK-Linux guide 

### PR DESCRIPTION
**Pull request by GeorgePostolakiy**
Fixed link and path for avrocpp in C++/SDK Linux guide

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by vadimol** [comment]
Looks good for me

**Comment by Oldnickus** [comment]
@kaa-leeroy add to whitelist

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ Before building the C++ endpoint SDK, install the following components on your m
 4. Install the [AvroC++](http://avro.apache.org/docs/1.7.6/api/cpp/html/index.html) library manually:
 
    ```
-   wget http://archive.apache.org/dist/avro/avro-1.7.5/cpp/avro-cpp-1.7.5.tar.gz
+   wget https://archive.apache.org/dist/avro/avro-1.7.7/cpp/avro-cpp-1.7.7.tar.gz
    tar -zxf avro-cpp-1.7.7.tar.gz
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md]: Fix this line instead as we use avro-cpp-1.7.5 in all other guides, SDKs and server. Don't update version in the guide as it wasn't generally tested.

Comment (GeorgePostolakiy) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md]: done, thanks

**Comment by vadimol** [comment]
+

**Pull request merged**